### ISSUE TITLE
Favorite Area Suggestion

### ISSUE THREAD

**Issue by MysticStickly** [PokeNavBot/PokeNav#80]
Suggestion for a potential new feature:

One of the biggest pleasures in recruiting new members to a raiding discord is seeing their eyes light up from all the potential raids and Pokenav  features they now have access to.  Additionally, we frequently see new members who are completely new to our territories (have moved here from a new city, state, maybe even country).  But with new opportunities comes a significant and sometimes daunting question (especially for new members in very large discords); who should they add as in-game friends?

The feature we propose is an addition to the trainer profile setup.  We have a $favorite feature for trainers to sport their favorite pokemon; we suggest adding a $favorite-area feature.  Some trainers predominantly raid in one specific region. Maybe a trainer raids mostly in Landsdale in the Bucks-Mont area or King of Prussia in the Montco-Chesco area.  If a trainer updates their profile with a $favorite-area command, they will have their username added to an area listing.  Then, if a new member finds themselves raiding mostly or entirely in King of Prussia, for example, they can use a command such as $get-fav-area King of Prussia, to which Pokenav will provide a listing of every trainer who associates mostly with King of Prussia (a listing of those who said $favorite-area King of Prussia).  The benefit of a list feature such as this would be the constant accessibility for new (and established) trainers to find the most sensible users to add as friends, and subsequently use the $gtc feature to add those trainers 

Thank you for the consideration,
MontCo/ChesCo Regional PoGO

**Comment by MiichelleErin** [comment]
Thank you for putting this together, George @MysticStickly. 

Dan, this would be useful for larger servers that cover multiple or many areas. Another thought though is that the server could really just use Discord tags for this?

**Comment by dandesousa** [comment]
Yeah I like this idea, though I'm avoiding new features and focusing on bug fixes and performance stuff until I get back from vacation.


I think this is a good idea. The area response could list the trainers friend codes already. I think its a good feature. 

In Bucks-Mont we use area tags (in-horsham, in-hatboro, in-ambler, in-abington) for this and another bot to list the members of the role. We don't put a focus on the social aspect of it. If you have a moderately sized list, this tends to work pretty well. This might actually get you off the ground sooner until get time to work on this in August. 

I'll circle back with you guys with my ideas on the feature before I get into the weeds.
